docs(review): docs content review (#2703)
scaleway/docs-content: managed-databases/document-database/how-to/enable-autobackup.mdx

  paragraph: This page explains how to enable autobackup for Databases
tags: managed-dabatase ferretdb autobackup database document
dates:
  validation: 2024-02-08
  posted: 2023-08-01
categories:
  - managed-databases
---

> With the autobackup feature, the creation and management of your Database Instance snapshots is automated according to your autobackup schedule configuration.

Reviewer: ```suggestion
With the autobackup feature, the creation and management of your Database Instance snapshots is automated according to your schedule configuration.
```

</Message>

<Message type="note">

>   The autobackup feature is enabled by default upon the creation of a Database Instance. Simultaneously, a schedule policy is created. The default autobackup frequency is once per day with a retention period of 7 days. You can change this in the Database Instance information page later.

Reviewer: ```suggestion
  The autobackup feature is enabled by default upon the creation of a Database Instance. Simultaneously, a schedule policy is created. The default autobackup frequency is once per day, with a retention period of 7 days. You can change this in the Database Instance information page later.
```

  <br /><br />
  By default, the Database Instance creation date is used as the base timestamp for the automated snapshots. When you change the schedule configuration, the time you validated the new configuration becomes the new base timestamp. For example, if at 8 a.m. on a Thursday you update the autobackup frequency to 24h, your next autobackup will be created at 8 a.m. on a Friday.
</Message>

1. Click **Document Databases** under **Managed Databases** on the side menu. A list of your Database Instances displays.
2. Click the Database Instance name or <Icon name="more" /> > **More info** to access the Database Instance information page.
3. Click <Icon name="edit" /> in the **Snapshots** tab to configure the backup frequency and retention.
    <Message type="note">

>       Although autobackup is enabled by default, you can disable the feature before you create the Instance, by unchecking the box in the creation wizard. You can also disable it after the Database Instance is ready.

Reviewer: ```suggestion
      Although autobackup is enabled by default, you can disable the feature before creating an Instance, by unchecking the box in the creation wizard. You can also disable it after the Database Instance is ready.
```
